tests/nodes/NodeConnection.test.ts

```diff
@@ -733,7 +733,7 @@ describe(`${NodeConnection.name} test`, () => {
         | NodeConnection<grpcTestUtils.GRPCClientTest>
         | undefined;
       let testProxy: Proxy | undefined;
-      let testProcess: child_process.ChildProcessWithoutNullStreams | undefined;
+      let testProcess: childProcess.ChildProcessWithoutNullStreams | undefined;
       try {
         const testProcess = spawnFile('tests/grpc/utils/testServer.ts');
```

Comment:
I'm wondering what the purpose of these are? Can these not be done through our other exec utilities? Like `pkExec` now that it's for generic executions? Why not just directly call `childProcess.spawn`. This utility seems useless.

Reply:
Use `import type { ... } from 'child_process';` here instead. That way no confusion with using `childProcess` directly.

Reply:
Done